src/Mentat.Repository/Services/CardService.cs

```diff
@@ -35,9 +30,38 @@ public void RemoveCard(string id)
             _cards.DeleteOne(card => card.Id.Equals(id));
         }
 
-        public void UpdateCard(string id, Card card)
+        public void SaveCard(string id, IFormCollection collection)
+        {
+            var card = new Card();
+            if (id != null)
+            {
+                card = GetCard(id);
```

Comment:
Please review my PR referenced in another comment or the official Mongo dotnet api on the methods InsertOne and ReplaceOne. Those methods give back return objects that will tell you if it was successful or not. There is no reason to check if the card is in the database before running these insert and update operations and these are wasted calls.

Reply:
I agree with Jonny. Take advantage of the return objects so that you can make your insert/update calls "fire-and-forget".

Reply:
@dyohn, so this was code that was not written by me, but I did relocate it. I kept the check because, I would think we would want to fail on an edit, if the ID given to us (via the view or something like Postman) was not in our DB.
Any thoughts?

Reply:
Okay, I think that's reasonable. It can stay as-is for now.

Reply:
@dyohn gotcha, thank you!